client/landing/stepper/declarative-flow/internals/steps-repository/error-step/index.tsx

```diff
@@ -25,12 +25,12 @@ const ErrorStep: Step = function ErrorStep( { navigation } ) {
 		domain = siteDomains[ 0 ].domain;
 	}
 
-	const headerText = siteSetupError ? siteSetupError.error : __( "We've hit a snag" );
```

Comment:
This kind of syntax changes should be handled in a separate PR. That way the actual feature being implemented is easier to follow 👍